models/warehouse/warehouse.sdf

```diff
@@ -0,0 +1,437 @@
+<?xml version="1.0" ?>
```

Comment:
where is this different from the existent warehouse?

Reply:
This is the model as specified in the warehouse.world, it should follow the convention of the other worlds where it is included as a model not just raw pasted SDF into a world file.